pkg/data/queries.go

```diff
@@ -0,0 +1,26 @@
+package data
+
+const addCourseQuery = `INSERT INTO "courses" ("code","title","department_code","description") VALUES ($1,$2,$3,$4)`
```

Comment:
could we join them instead of multiple writing `const`

Use the following feature:
```
const (
    addCourseQuery = ...
    readAllCoursesQuery = ...
   etc....
)
```